doc/src/records/accumulator.md

```diff
@@ -6,11 +6,27 @@ TODO: expand description and example
 
 ACCUMULATOR results must be reported using user-defined REPORTs or EXPORTs.  For example --
 
-    REPORT rpType=UDT rpFreq=Month rpDayBeg=Jan 1 rpDayEnd=Dec 31
-        REPORTCOL colHead="mon" colVal=$Month colWid=3
-        REPORTCOL colHead="Total" colVal=@Accumulator[ 1].M.acmSum colDec=0 colWid=10
-        REPORTCOL colHead="Average" colVal=@Accumulator[ 1].M.acmMean colDec=0 colWid=10
+    ACCUMULATOR "Zone Heating Set Point"
+      acmValue = @Zone[ 1].znTH // Subhourly output
+
 
+    REPORT rpType=UDT rpFreq=Month rpDayBeg=Jan 1 rpDayEnd=Dec 31
+        REPORTCOL colHead="Month"           colVal=$Month                                                          colWid= 3
+        REPORTCOL colHead="Monthly Total"   colVal=@Accumulator["Zone Heating Set Point"].M.acmSum  colDec=0 colWid=10
```

Comment:
I'm not sure anyone would ever want the sum of a temperature output. Maybe we can find a different probe to use as an example? Maybe something like unmet load hours?